plugin/mutate/contributors.md

```diff
@@ -0,0 +1,15 @@
+# Notable contributions
+
+The following people deserves recognition as a result of their help and inspiration in the development and implementation of dextool mutate.
+
+- [Sten Vercammen](https://github.com/Sten-Vercammen)
+- [John Tinnerholm](https://github.com/JKRT)
+- [John Törnblom](https://github.com/john-tornblom)
+- [Eric Lindskog](https://github.com/EricLindskog) & [Lisa Habbe](https://github.com/lisahabbe)
+- [Axel Boström](https://github.com/axelbostrom), [Oliver Green](https://github.com/OliverGreen27) & [Viktor Norgren](https://github.com/virrenn)
```

Comment:
Missing your name

Reply:
Yeah maybe, but I won't add myself to the list of "notable contributors". That feels very much like patting myself on the back.

Reply:
then i'll pat you